appyters/Gene_Expression_by_Tissue/Gene_Expression_by_Tissue.ipynb

```diff
@@ -138,7 +138,33 @@
     "from IPython.display import HTML, display, Markdown\n",
     "from maayanlab_bioinformatics.harmonization.ncbi_genes import ncbi_genes_lookup\n",
     "from maayanlab_bioinformatics.harmonization.transcripts import transcripts_to_genes\n",
-    "lookup = ncbi_genes_lookup()\n"
+    "import itertools as IT\n",
+    "\n",
+    "lookup = ncbi_genes_lookup()\n",
+    "\n",
+    "def find_gene(chunks):\n",
```

Comment:
One potential issue with your change is in the rare case that the background maps multiple values to a single gene. I think it will be better to use `index_col=[0, 1]` as before then `find_gene` becomes (please test):

```python
def find_gene(chunks, gene): # also recommend you add gene as parameter to this function
  data = [
    chunk[chunk.index.get_level_values(0).map(lambda x: lookup(x.split('.')[0])) == gene]
    for chunk in chunks
  ]
  return pd.concat(data).median(level=1)
```

Reply:
Good catch, didn't think about that case. I've changed it to what you suggested and tested it.